spec/index.bs

```diff
@@ -4723,6 +4723,20 @@ must be well nested.
 
             **Returns:** {{undefined}}
 
+            If any of the following conditions are unsatisfied, generate a {{GPUValidationError}} and stop.
+            <div class=validusage>
+                - |this|.{{GPUCommandEncoder/[[state]]}} is {{encoder state/open}}.
+                - |querySet| is [$valid to use with$] |this|.
+                - |destination| is [$valid to use with$] |this|.
+                - |destination|.{{GPUBuffer/[[usage]]}} contains {{GPUBufferUsage/QUERY_RESOLVE}}.
+                - |firstQuery| is less than the number of queries in |querySet|.
+                - (|firstQuery| + |queryCount|) is less than or equal to the number of queries in |querySet|.
+                - The queries between [|firstQuery|, |firstQuery| + |queryCount| - 1] are written by query operations.
+                - |destinationOffset| is a multiple of 8.
+                - |destinationOffset| is less than or equal to |destination|.{{GPUBuffer/[[size]]}}.
+                - The size of the |queryCount| of queries is less than or equal to (|destination|.{{GPUBuffer/[[size]]}} - |destinationOffset|).
```

Comment:
These two can be written as a single rule, as these are Real numbers that can't overflow.
```suggestion
                - |destinationOffset| + 8 &times; |queryCount| &le; |destination|.{{GPUBuffer/[[size]]}}.
```